### PR TITLE
feat: update to GA provider for Cloud Identity resources

### PR DESCRIPTION
**Pull request by xingao267**

### CODE REVIEW

File: versions.tf [hunk 1]
```diff
@@ -17,7 +17,6 @@
 terraform {
   required_version = ">=0.12.6"
   required_providers {
-    google      = ">= 3.29, <4.0.0"
-    google-beta = ">= 3.29, <4.0.0"
+    google = ">= 3.49, <4.0.0"
```
Comment (umairidris) [inline versions.tf:20]: I believe this is normally considered a breaking change, but I think it should be ok given the recency of the module and the relatively minor bump in version. I'll let @bharathkkb confirm.

**Review by umairidris — APPROVED** [review]

**Comment by xingao267** [comment]
I think there are still features such as security groups that are in beta, which we need. I'll hold on this PR for future. Closing for now.

**Pull request closed without merging**